selfdrive/ui/qt/device/device.cc

```diff
@@ -0,0 +1,82 @@
+#include "selfdrive/ui/qt/device/device.h"
```

Comment:
there's too much nesting here, no need for a device folder

Reply:
@adeebshihadeh Fixed.